SideCiSample7.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class SideCiSample7 {
+public function hoge() { echo "hoge"; }
```

Comment:
__[code_sniffer]__
* Line indented incorrectly; expected at least 4 spaces, found 0
* Opening brace should be on a new line
* Opening brace must be the last content on the line
* Closing brace must be on a line by itself